presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -399,6 +394,8 @@ public void testFilesForAbortedTransactionsIgnored()
             QueryResult onePartitionQueryResult = query(selectFromOnePartitionsSql);
             assertThat(onePartitionQueryResult).containsExactly(row(1), row(2));
 
+            String tableLocation = getTablePath(tableName, 1); // pass 1 to trim /delta_... suffix
```

Comment:
can `getTablePath` be smart enough for that?

Reply:
AC as https://github.com/prestosql/presto/pull/5442/commits/a2e18711bc4b48293e1a04e8e7bcce9ce66af894